CODEOWNERS

```
@@ -195,6 +197,8 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 /*/extensions/matching/input_matchers/ip @aguinet @snowp
 # Key Value store
 /*/extensions/key_value @alyssawilk @ryantheoptimist
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @dio @zuercher
```

Sorry, this one as well.

```suggestion
/*/extensions/http/stateful_session/cookie @wbpcode @dio
```

---

sorry, it my negligence.